src/seedu/addressbook/commands/Command.java

```diff
@@ -36,10 +37,23 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
 
+
+
```

Comment:
Try to avoid these stray spaces. May need to configure your IDE to not include these line breaks.